Source/SwiftLintFramework/Extensions/String+SwiftLint.swift

```diff
@@ -69,4 +69,16 @@ extension String {
         }
         return nil
     }
+
+    var stringByDeletingLastPathComponent: String {
```

Comment:
That motivation makes a lot of sense to me. My thinking was mostly DRY, and how it would make it trivial to swap this out project-wide with a Swift `String` native approach if desired. But I'd be happy to revert.